README.md

```diff
@@ -76,6 +87,13 @@ the variable pizza inside your method.
 Your Answer:
 ```js
 // write code here
+var pizza = {
+  temperature: 70,
+  bake: function(){
+    this.temperature = 300;
+    return(this.temperature);
+  }
+}
 ```
```

Comment:
:+1: Although in this case you do not need to return `this.temperature` in `bake`.